packages/client/components/ui/card/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { ReactNode } from 'react'
 import styles from './card.module.scss'
 import classNames from 'classnames'
-import tinycolor, { mostReadable } from 'tinycolor2'
+import Tinycolor from 'tinycolor2'
```

Comment:
Why is it uppercase?

Reply:
Because I am importing the class instead of the contructor function. Something seems to be wrong with their exports (or our setup), which is why just importing and calling the function doesn't work.